interfaces/tls_certificates/v1/schema.py

```diff
@@ -0,0 +1,101 @@
+"""
+This file defines the schemas for the provider and requirer sides of the `tls_certificates` interface.
+
+It exposes two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+
+Examples:
+    ProviderSchema:
+        unit: <empty>
+        app: {
+            "certificates": [
+                {
+                    "ca": "-----BEGIN CERTIFICATE----- ...",
+                    "chain": [
+                        "-----BEGIN CERTIFICATE----- ...",
+                        "-----BEGIN CERTIFICATE----- ..."
+                    ],
+                    "certificate_signing_request": "-----BEGIN CERTIFICATE REQUEST----- ...",
+                    "certificate": "-----BEGIN CERTIFICATE----- ..."
+                }
+            ]
+        }
+    RequirerSchema:
+        unit: {
+            "certificate_signing_requests": [
+                {
+                    "certificate_signing_request": "-----BEGIN CERTIFICATE REQUEST----- ...",
+                    "ca": true
+                }
+            ]
+        }
+        app:  <empty>    
+"""
+
+from typing import List, Optional
+from pydantic import BaseModel, Field, Json
+from interface_tester.schema_base import DataBagSchema
+
+
+class Certificate(BaseModel):
+    """Certificate model."""
+    ca: str = Field(
+        description="Whether the certificate is a CA."
```

Comment:
I think this description is wrong, and it should be the actual certificate?

I know these names probably can't change without breaking the interface, but using `is_ca` for the other, or `ca_cert` here would be helpful to avoid confusion. Possibly something to keep in mind for future iterations.